rancher2/resource_rancher2_auth_config_githubapp.go

```diff
@@ -0,0 +1,120 @@
+package rancher2
+
+import (
+	"fmt"
+	"log"
+
+	"github.com/hashicorp/terraform-plugin-sdk/helper/schema"
+	managementClient "github.com/rancher/rancher/pkg/client/generated/management/v3"
+)
+
+func resourceRancher2AuthConfigGithubApp() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceRancher2AuthConfigGithubAppCreate,
+		Read:   resourceRancher2AuthConfigGithubAppRead,
+		Update: resourceRancher2AuthConfigGithubAppUpdate,
+		Delete: resourceRancher2AuthConfigGithubAppDelete,
+		Schema: authConfigGithubAppFields(),
+	}
+}
+
+func resourceRancher2AuthConfigGithubAppCreate(d *schema.ResourceData, meta interface{}) error {
+	client, err := meta.(*Config).ManagementClient()
+	if err != nil {
+		return fmt.Errorf("[ERROR] Failed to get ManagementClient %s", err)
```

Comment:
Here and everywhere else, though may not matter much, we should probably favor `%w` verb over `%s` when returning the error in case it's needed to be used in any meaningful way.

Reply:
I totally agree, and those ugly `[ERROR]` elements...but this is consistent with the other resources.

This code will be replaced pretty soon with the new Terraform SDK https://github.com/rancher/terraform-provider-rancher2/pull/1697 and hopefully the "standard" will improve :-)